build/index.html

```diff
@@ -821,6 +832,16 @@ <h5 class="head-5 text--white margin-bottom-medium">Update a customer</h5>
       </td>
       <td>The customer&rsquo;s full name (first and last).</td>
     </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">password</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+        This must be at least eight characters long. Omitting this field will prompt the customer to set their password the next time they sign in.
```

Comment:
`Omitting this field will prompt the customer to set their password the next time they sign in.` This isn't the current behavior. The password is optional and won't be modified unless it is passed in, but does not cause the customer to be prompted to set their password on next login. Is that the functionality we want?

Reply:
Another thing I noticed is that the Customer update docs are incorrect in that you cannot modify the plan via that endpoint. I'm guessing it was incorrectly copied over from the Customer create portion of the docs.